spec/commands/edit_spec.rb

```diff
@@ -186,15 +182,12 @@ def last_exception; @pry.last_exception; end
 
       after do
         @tf.close(true)
-        File.unlink("#{@path}c") if File.exist?("#{@path}c") #rbx
+        File.unlink("#{@path}c") if File.exist?("#{@path}c")
```

Comment:
You could just get rid of this line I think

Reply:
Looks like you're right. For some reason when I had tried that it didn't work and some tests were failing.